qt/widgets/instrumentview/src/Viewport.cpp

```diff
@@ -310,6 +322,29 @@ void Viewport::setViewToZNegative() {
   Mantid::Kernel::Quat tempy(180.0, Mantid::Kernel::V3D(0.0, 1.0, 0.0));
   m_quaternion = tempy;
   m_quaternion.GLMatrix(&m_rotationmatrix[0]);
+  adjustProjection();
+}
+
+void Viewport::adjustProjection() {
+  // reset the projection bounds to the original values
+  m_left = m_leftOrig;
+  m_right = m_rightOrig;
+  m_top = m_topOrig;
+  m_bottom = m_bottomOrig;
+  m_near = 0.0;
+  m_far = 0.0;
+  // rotate the original projection based on the new quaternion
+  m_quaternion.rotateBB(m_left, m_bottom, m_near, m_right, m_top, m_far);
+  // restore the Z bounds
+  m_near = m_nearOrig;
+  m_far = m_farOrig;
+  // update the GL projection with the new bounds
+  applyProjection();
+}
+
+void Viewport::getProjection(Mantid::Kernel::V3D &minBound, Mantid::Kernel::V3D &maxBound) const {
```

Comment:
Is this used?

Reply:
No, sorry about that, I just removed it. I put it in for accessing the projection bounds when drawing the bounding boxes to screen, but it is not needed anymore.